article/app/views/fragments/adSlotFallback.scala.html

```diff
@@ -0,0 +1,18 @@
+<div class="dumathoin" data-component="dumathoin-mpu-fallback">
+  <p class="dumathoin__copy">
+    Advertising helps pay for our journalism ...Can you support us another way?
+  </p>
+  <p class="dumathoin__copy">
+    For less than the price of a coffee a week, you could help secure the Guardian’s future.
+  </p>
+  <a class="dumathoin__button" data-link-name="dumathoin supporter button"
+    href="https://membership.theguardian.com/supporter?INTCMP=mpucontainer" target="_blank">
+    <span class="dumathoin__button-text">Become a Supporter</span>
+    @fragments.inlineSvg("arrow-right", "icon", List("arrow-right"))
+  </a>
+  <a class="dumathoin__button" data-link-name="dumathoin contribution button"
+    href="https://contribute.theguardian.com?INTCMP=mpucontainer" target="_blank">
```

Comment:
It's not about trust, but about reducing attack vectors in case one day somewhere something goes wrong :)